app/assets/stylesheets/admin/admin.sass

```diff
@@ -0,0 +1,47 @@
+// -----------------------------------------------------------------------------
+// This file contains all Admin styles.
+// -----------------------------------------------------------------------------
+
+@import "normalize-rails"
+@import "bourbon"
+
+/*!
+ * Author:       The Frontier Group
+ * Author URI:   http://www.thefrontiergroup.com.au/
+ */
```

Comment:
Yes (1 and No (2

they have different purposes and behaviour:

Comment on L1 doesn't get output (it's silent) and describes the file / its contents
Comment on L8 is a "PSA" and will appear in the compiled stylesheet – the `/*!` causes that

>This is useful for adding Copyright notices to your generated CSS.

See [Sass Reference](http://sass-lang.com/documentation/file.SASS_REFERENCE.html#comments)